mailpoet/assets/js/src/automation/editor/components/filters/chip.tsx

```diff
@@ -39,15 +42,15 @@ export function StepFilters({ step }: Props): JSX.Element | null {
           onClick={onToggle}
           ariaExpanded={isOpen}
         >
-          {__(`Filters: ${filterCount}`, 'mailpoet')}
+          {__(`${strings.label}: ${filterCount}`, 'mailpoet')}
```

Comment:
Shouldn't it be `sprintf(__('Filters: %d', 'mailpoet'), filterCount)`?

Reply:
Good catch! The translation is actually not needed here at all, the strings should come in translated. Fix: https://github.com/mailpoet/mailpoet/pull/5216/commits/5799461827702d7aac8c1f6394adc0050f300e30

Reply:
Not sure if there are edge cases in translations where something like `%d: TranslatedLabel` would be the correct order of the elements. I don't think it's in LTR languages, but not sure if there are other cases maybe. This is why I would suggest something like `sprintf(strings.labelCount, filterCount)` and adding to the strings `labelCount: __('Filters: %d', 'mailpoet')`. But up to you, not super nice as well.

Reply:
:+1: Makes sense, https://github.com/mailpoet/mailpoet/pull/5216/commits/e5dacda93c4354c1e4daf62147fd5f7cb3239e12 + https://github.com/mailpoet/mailpoet/pull/5216/commits/569f83a80295d63a09d9daa8cc219aeb7c86bc8b.